josh-json/src/files.js

```diff
@@ -14,7 +14,7 @@ class FileManager {
     }
   }
   async has(key, path) {
-    const index = await this.getFile('index.json');
+    const index = await this.getIndex();
```

Comment:
this was the only change haha, getIndex does some checks so if the db was empty then it would fail